Branch-SDK-TestBed/src/io/branch/branchandroiddemo/MainActivity.java

```diff
@@ -24,6 +25,17 @@
 
 public class MainActivity extends Activity {
 	Branch branch;
+
+    public enum SESSION_MANAGEMENT_MODE {
```

Comment:
Added this just for switching between Session Management mode while we are testing.SDK already know about the mode by checking whether application is an instance of extending BranchLinkApp class.The following code in SDK is checking for it .
''' public static void createInstance(Application application) {
		isBranchAppInstantiated = application instanceof  BranchLinkApp; '''